{
  "id": "MAL-2024-6260",
  "summary": "Malicious code in zproxy (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "aliases": [
    "SNYK-PYTHON-ZPROXY-5295974"
  ],
  "modified": "2024-06-28T03:15:20.971768Z",
  "published": "2024-06-25T13:44:31Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:44:31Z",
        "import_time": "2024-06-28T02:51:15.479393402Z",
        "versions": [
          "1.0"
        ],
        "id": "RLMA-2024-05064",
        "source": "reversing-labs",
        "sha256": "d71c3dff3b92802aa1cb79fdf039810ba8aa7b96aa3678d4c7548dae604d95ac"
      }
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-ZPROXY-5295974"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "zproxy",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/zproxy"
      },
      "versions": [
        "1.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/zproxy/MAL-2024-6260.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}